src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -330,6 +330,38 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}:another")
     }
 
+    def 'tags are correctly processed'() {
```

Comment:
Can you just test the 'computeName' method separately? No need for end-to-ebd here.